src/components/header.tsx

```diff
@@ -465,6 +495,18 @@ const Header = ({
           <NonMobileMenu style={headerProps.wide ? { paddingRight: '200px' } : {}}>
             <SecondLevelMenu />
           </NonMobileMenu>
+          <GithubLink href="https://github.com/prisma" target="_blank">
+            <Github className="light" fill={'#2d3748'} width={24} height={24} />
+            <Github className="dark" fill={'#ffffff'} width={24} height={24} />
+          </GithubLink>
+          <Button
+            type="primary"
+            color="teal"
+            className="log-btn"
+            href="https://console.prisma.io/login?utm_source=website&utm_medium=header"
```

Comment:
use this link please: https://console.prisma.io/login?utm_source=docs&utm_medium=login